### PR TITLE
Revert "[Dialogs] Update buttons touch area to be 48x48 minimum (#4624)"

### PR DESCRIPTION
**Pull request by ianegordon**
This reverts commit 2c8539c3be9bb39dd8198341fb71c860cbc61e93.

There were some issues with the vertical layout that need to be addressed.

### CODE REVIEW

**Review by romoore — APPROVED** [review]
LGTM

**Review by codeman7 — APPROVED** [review]

**Comment by romoore** [comment]
Comment originally from #4624 (post-submit)

> Looks like this introduced overlapping button touch targets in the "Overpopulated" Alert example.  Here's a print-out of the 6 vertically-stacked buttons. For each, you can see that the vertical touch targets overlap by 4 points.

> Button 1: 2..50
Button 2: 46..94
Button 3: 90..138
Button 4: 134..182
Button 5: 178..226
Button 6: 222..270

```
2018-07-30 10:39:19.831 MDCCatalog[19231:215705] Button (0x79d81ee0) touch area: {{102.5, 2}, {75, 48}}
2018-07-30 10:39:19.831 MDCCatalog[19231:215705] Button (0x79f79720) touch area: {{102.5, 46}, {75, 48}}
2018-07-30 10:39:19.832 MDCCatalog[19231:215705] Button (0x79f641d0) touch area: {{102.5, 90}, {75, 48}}
2018-07-30 10:39:19.832 MDCCatalog[19231:215705] Button (0x79f701b0) touch area: {{102.5, 134}, {75, 48}}
2018-07-30 10:39:19.832 MDCCatalog[19231:215705] Button (0x79fb0340) touch area: {{102.5, 178}, {75, 48}}
2018-07-30 10:39:19.832 MDCCatalog[19231:215705] Button (0x79faa5f0) touch area: {{113.5, 222}, {53, 48}}
```

File: components/Dialogs/src/private/MDCAlertControllerView+Private.m [hunk 1]
```diff
@@ -35,7 +35,6 @@
 static const CGFloat MDCDialogActionsVerticalPadding = 8.0;
```
Comment (codeman7) [inline components/Dialogs/src/private/MDCAlertControllerView+Private.m:35]: Should this just be updated to 12 to fix the touch area and the overlapping issue?

**Review by codeman7 — APPROVED** [review]

File: components/Dialogs/src/private/MDCAlertControllerView+Private.m (hunk 1 above)
Comment (ianegordon) [reply components/Dialogs/src/private/MDCAlertControllerView+Private.m]: Let's roll-back and we can verify the increased padding solves the issue.
In general, the philosophy is roll-back immediately, then fix.

Comment (ianegordon) [reply components/Dialogs/src/private/MDCAlertControllerView+Private.m:35]: Let's rollback so we can verify the fix.  We definitely don't want this to go out in the release.

**Pull request merged**